alpha/lib/model/assetPeer.php

```diff
@@ -825,6 +825,7 @@ public static function retrieveAudioFlavorsByEntryID($entryId)
 		$c = new Criteria();
 		$c->add(assetPeer::ENTRY_ID, $entryId);
 		$c->add(assetPeer::IS_ORIGINAL, 0);
+		$c->add(assetPeer::STATUS, asset::ASSET_STATUS_READY);
```

Comment:
Since this function can be use to get audio in all status, add the status as optional in the function input.
retrieveAudioFlavorsByEntryID($entryId, $statusList = array())
and add it as Status in.

The caller can add the ready status.